File: apps/docs/src/code/python/fs_ls.py
import asyncio
from e2b import Session

async def main():
  session = await Session.create(id="Nodejs")

  # List directory
  dir_content = await session.filesystem.list('/')
  for item in dir_content:
    print(item)

  await session.close()

asyncio.new_event_loop().run_until_complete(main())

File: apps/web/src/code/js/basics/fs_ls.js
import { Sandbox } from 'e2b'

const sandbox = await Sandbox.create({
  template: 'base',
})

const dirContent = await sandbox.filesystem.list('/') // $HighlightLine
dirContent.forEach((item) => {
  console.log(item.name)
})

await sandbox.close()
